uptane/clients/primary.py

```diff
@@ -952,8 +952,8 @@ def register_ecu_manifest(
     tuf.formats.BOOLEAN_SCHEMA.check_match(force_pydict)
 
     if vin != self.vin:
-      raise uptane.Error('Received an ECU Manifest supposedly hailing from a '
-          'different vehicle....')
+      raise uptane.UnknownVehicle('Received an ECU Manifest supposedly hailing '
```

Comment:
This change has not been reflected in the docstring, it still indicates that `uptane.Error` is raised for an unknown VIN:

https://github.com/awwad/uptane/blob/c56146774048d80208332b868d1076ba23786262/uptane/clients/primary.py#L944-L948

Reply:
Sorry, I meant to comment on this: It's not changed in the docstring in this PR because it's changed in #136, a simultaneous PR that modified the docstring more thoroughly. When I split the commits into discrete PRs, it was too much of a hassle to restrict merge order or allow a conflict, so I just left that change in that PR.